types/peers.pp

```diff
@@ -0,0 +1,8 @@
+type Wireguard::Peers = Array[
+  Struct[{
+    public_key           => String[1],
+    allowed_ips          => Optional[Array[String]],
+    endpoint             => Optional[Variant[String[1],Stdlib::IP::Address::V4::CIDR,Stdlib::IP::Address::V6::CIDR]],
```

Comment:
If you allow `String[1]`, doesn't that already include all IPs? You wouldn't get any additional type safety
```suggestion
    endpoint             => Optional[String[1]],
```

Reply:
you are right